src/PrestaShopBundle/Controller/Admin/Sell/Address/AddressController.php

```diff
@@ -473,6 +476,99 @@ public function editOrderAddressAction(int $orderId, string $addressType, Reques
         ]);
     }
 
+    /**
+     * Handles edit form rendering and submission for cart address
+     *
+     * @AdminSecurity(
+     *     "is_granted('update', request.get('_legacy_controller'))",
```

Comment:
If I am not wrong, it means `admin_cart_address_edit` route can only be used in "Create an order" page if BO user is granted "Adresses" rights 🤔  isn't it a bit strange ? Maybe we should rather require Order rights here

Reply:
In a way it makes sense.. You can select the address in the order based on the existing ones But if you want to edit the addresses you must have the permission for it No?

Reply:
Yeah, indeed with this point of view it could make sense.

Let's keep it like this for now and ask @PrestaShop/prestashop-product-team for the long term 👍